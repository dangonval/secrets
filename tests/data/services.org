#+NAME:services
|----------+-------+----------+-----------+---------+----------------------+----------+---------+---------------+--------+------------+------------------+-------+------------------+-----------+-----------------+------------|
| company  | owner | customer | tags      | country | url                  | name     | created | updated       | status | usernameId | userDescription  | pwdId | pwdDescriptions  | questions | description     | dateChange |
|----------+-------+----------+-----------+---------+----------------------+----------+---------+---------------+--------+------------+------------------+-------+------------------+-----------+-----------------+------------|
| company1 |       |          | tag1,tag2 |         | http://something.com | Service1 |         | 20. Jun. 2011 |        | AA01       | Some description | XX01  | Some description | Q01:AW01  | someDescription |            |
| company2 |       |          |           |         | http://whaever.com   | Service2 |         |               |        | AA02       | Some description | XX02  |                  | Q02:AW02  | someDescription |            |
|----------+-------+----------+-----------+---------+----------------------+----------+---------+---------------+--------+------------+------------------+-------+------------------+-----------+-----------------+------------|

#+NAME:questions
|-----+----------------|
| id  | question       |
|-----+----------------|
| Q01 | Are you there? |
| Q02 | Is this blue?  |
|-----+----------------|

#+NAME:descriptions
|-------+-------------|
| pwdId | description |
|-------+-------------|
| XX02  | Whatever    |
|-------+-------------|
