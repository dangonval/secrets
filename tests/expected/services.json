[
  {
    "company": "company1",
    "owner": "",
    "customer": "",
    "tags": "tag1,tag2",
    "country": "",
    "url": "http://something.com",
    "name": "Service1",
    "created": "",
    "updated": "20. Jun. 2011",
    "status": "",
    "usernameId": "AA01",
    "userDescription": "Some description",
    "pwdId": "XX01",
    "pwdDescriptions": "Some description",
    "questions": [
      {
        "question": "Are you there?",
        "answerId": "AW01",
        "answer": "AW01"
      }
    ],
    "description": "someDescription",
    "dateChange": "",
    "username": "AA01",
    "secretPasswords": [
      {
        "passwordId": "XX01",
        "password": "XX01",
        "description": "Some description"
      }
    ]
  },
  {
    "company": "company2",
    "owner": "",
    "customer": "",
    "tags": "",
    "country": "",
    "url": "http://whaever.com",
    "name": "Service2",
    "created": "",
    "updated": "",
    "status": "",
    "usernameId": "AA02",
    "userDescription": "Some description",
    "pwdId": "XX02",
    "pwdDescriptions": "",
    "questions": [
      {
        "question": "Is this blue?",
        "answerId": "AW02",
        "answer": "AW02"
      }
    ],
    "description": "someDescription",
    "dateChange": "",
    "username": "AA02",
    "secretPasswords": [
      {
        "passwordId": "XX02",
        "password": "XX02",
        "description": "Whatever"
      }
    ]
  }
]
